tests/test_cache_invalidation.py

```diff
@@ -5,8 +5,13 @@
 # the Apache 2.0 License: http://www.apache.org/licenses/LICENSE-2.0
 
 
-import asyncpg
 from asyncpg import _testbase as tb
+from asyncpg.exceptions import InterfaceError, InvalidCachedStatementError
+from asyncpg.exceptions import OutdatedSchemaCacheError
```

Comment:
We allow only module imports in `*.py` files.  Access exceptions like `asyncpg. OutdatedSchemaCacheError`

Reply:
It is a test.
Moreover the `test_connect.py` has
```python
from asyncpg.serverversion import split_server_version_string
```
(khm, ok, the above is from one of my commit, but there were no nitpickings)

the `test_record.py` has
```python
from asyncpg.protocol.protocol import _create_record as Record
```
(that one is written by you in the 487f0f17e148ba6b4bfb7f03a438bb8ba02650a1)
---
In fact the only reason why I used it that way was long rows with `self.assertRaisesRegex(…)`.

OK, I've just changed it to fit your requirements.